docs/builds-releases.md

```diff
@@ -1,23 +1,49 @@
-As discussed in our [architectural overview](architecture.md), TinyStacks uses two YAML files ingested by AWS CodeBuild to manage your builds and releases.
+Deploying your application on TinyStacks is as easy as checking a change into source control. Behind the scenes, the process is divided into three parts: 
 
-## build.yml
+* **Build**. Compiles your new container containing your application.
```

Comment:
'container image' instead of 'container'?

Reply:
Good point - fixing.